### DISCUSSION TITLE
Copy/paste behavior under Linux (MATE on MX)

### DISCUSSION THREAD

**Discussion by jebofponderworthy** [ezra-bible-app/ezra-bible-app#940] [Q&A]
I am overjoyed to see this development and community.  I have done a certain amount of coding, am probably better suited for debugging, and would like very much to help.  

My desktop runs MX Linux (Debian-based, extremely stable), desktop being MATE.  I have not been able to consistently copy to clipboard more than one verse at a time.  Is there a GUI behavior I haven't figured out?  Or an Electron+MATE issue of some sort?  Anything I can do to help?

**Comment by tobias-klein** [comment]
Hi @jebofponderworthy 

Thanks for your question! I have just checked and could not reproduce the bug you described.
On my end I first select multiple verses using drag & drop or alternatively using ctrl + click on the verses I would like to select. Then I copy to clipboard using ctrl + c. Pasting based on that works reliably.

How did you select the verses? Did you use drag & drop or ctrl + click?

**Comment by mochsner** [comment]
Just wanted to note I've been able to copy on latest with Ubuntu 20.04/22.04. Notably, I'm only able to use the CTRL+V clipboard, not the CTRL+INSERT one (I forget the name)
